src/BrightScriptCommands.ts

```diff
@@ -263,6 +263,24 @@ export class BrightScriptCommands {
             }
         });
 
+        this.registerCommand('openQuickInputBox', async (url: string, placeholder: string, appNames: vscode.QuickPickItem[]) => {
```

Comment:
This command should have a more meaningful name. 

Also, I think it should just fetch the app data itself rather than needing them to be passed in. So something like this:
```typescript
this.registerCommand('openRegistry', async (host: string) => {
    if (!host) {
        host = await this.promptForHost();
    }
    
     const apps = await util.httpGet(`${host}:8060/query/apps`);
      //convert the apps to QuickPickItems
      
      //show the quickPick
      
      //open the URL if all is good

Reply:
Yeah, I think the quick pick is already simple enough when used directly from the vscode APIs, so I think it's more valuable to make this command specific to opening the registry.

Reply:
done